LICENSE

```diff
@@ -252,6 +252,16 @@ OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
 NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
 EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+Examples' Data Files License
+============================
+
+The files 'gdp_per_capita.csv' and 'country_codes.csv' were adapted from
+the World Bank Data Catalog.
+The World Bank: GDP per capita (current US$).
```

Comment:
This isn't descriptive enough for me to find the data set (by typing that name into http://datacatalog.worldbank.org/). The format should be `The World Bank: Dataset name: Data source`, and I think it wouldn't hurt to put the URL for the dataset in parentheses.